cmd/aws-k8s-agent/main.go

```diff
@@ -45,15 +45,12 @@ func _main() int {
 		return 1
 	}
 
-	discoverController := k8sapi.NewController(kubeClient)
-	go discoverController.DiscoverLocalK8SPods()
-
```

Comment:
Getting rid of this, and just reading it [from the CRI socket](https://github.com/aws/amazon-vpc-cni-k8s/pull/972/files#diff-716bb85864c637ee87fb1ae9ca36fad5R276) will resolve #711  and replace #738 as well...